server/serverbase.js

```diff
@@ -104,10 +104,18 @@ exports.initialize2 = function(app) {
       }
     }
   }
+  function loadThumb() {
```

Comment:
It's OK to leave the folder-thumbs question for later.

On Sat, Jun 13, 2015 at 8:06 AM, Liu Xinan <notifications@github.com> wrote:

> In server/serverbase.js
> <https://github.com/PencilCode/pencilcode/pull/132#discussion_r32370002>:
>
> > @@ -104,10 +104,18 @@ exports.initialize2 = function(app) {
> >        }
> >      }
> >    }
> > +  function loadThumb() {
>
> Are we gonna have thumbs for folders? If we are only to show the folder
> icon for folders, then calling loadThumb with a directory name should
> just return 404.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PencilCode/pencilcode/pull/132/files#r32370002>.
>